lib/webrick-patch.rb

```diff
@@ -0,0 +1,36 @@
+require 'webrick/https'
+
+CIPHERS = 'AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256'
```

Comment:
I assume this is based on https://wiki.mozilla.org/Security/Server_Side_TLS - ECDHE and DHE ciphers? Could you add a comment re: selection criteria here?